lib/yuriita/search_filter.rb

```diff
@@ -15,7 +15,7 @@ def apply(relation, keywords)
 
       combination.new(
         base_relation: relation,
-        relations: relations,
+        relations: relations
```

Comment:
Could this use the shorthand?
```suggestion
        relations:
```

Reply:
Nice catch! The `relations:` are now abridged in 6fc3d558eb64a80d243f4a8a44591b9dc8c4170b.